api/courses/1/practice.json

```diff
@@ -0,0 +1,99 @@
+{ "id": "Practice-Course-1",
+  "HACK_LOCAL_STEP_COMPLETION": true,
+  "title": "Chapter 5 and Chapter 6 Practice",
+  "description_html": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo. Proin sodales pulvinar tempor. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Nam fermentum, nulla luctus pharetra vulputate, felis tellus mollis orci, sed rhoncus sapien nunc eget odio.",
```

Comment:
There won't be any `description_html`in the practice task JSON.  If y'all need it for some reason, we should talk.